cmd/module.go

```
@@ -27,16 +31,37 @@ func showModulesCmd(ctx *config.Context) *cobra.Command {
 		Use:   "show",
 		Short: "Shows a list of modules included in the relayer",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			names := make([]string, len(ctx.Modules))
+			modules := make([]string, len(ctx.Modules))
+			bi, _ := debug.ReadBuildInfo()
```

nit: could you return the error if `ok` is false?